index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="./style.css">
+  <title>Document</title>
```

Comment:
Não era um requisito, mas ficaria muito bom um título bacana na página, exemplo:
```suggestion
  <title>Lições Aprendidas por Stanrley</title>
```

Reply:
Poxa, foi muito vacilo da minha parte, obrigado pelo toque!!!